dev-client/src/App.tsx

```diff
@@ -91,12 +91,12 @@ function App(): React.JSX.Element {
   return (
     <GestureHandlerRootView style={style}>
       <Provider store={store}>
-        <HeaderHeightContext.Provider value={{headerHeight, setHeaderHeight}}>
-          <BottomSheetModalProvider>
-            <PaperProvider theme={paperTheme}>
-              <NativeBaseProvider theme={theme}>
-                <Portal.Host>
-                  <NavigationContainer>
+        <NavigationContainer>
```

Comment:
The `BottomSheetScrollView` wasn't behaving correctly in the sites screen filters unless I moved the `NavigationContainer` to be above the outermost `BottomSheetModalProvider`.